From 442d0b5a0705a41395fbf5c1d195618f528325c3 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Mon, 9 Apr 2018 15:48:15 +0800
Subject: [PATCH 2/5] v4l: xilinx: csi2rxss: replace media_entity_init with
 media_entity_pads_init

This patch is to fix a compiling error according to the mainline commit ab22e77c
as follows:

drivers/media/platform/xilinx/xilinx-csi2rxss.c: In function 'xcsi2rxss_probe':
drivers/media/platform/xilinx/xilinx-csi2rxss.c:1638:8: error: implicit declaration of function 'media_entity_init';
|   ret = media_entity_init(&subdev->entity, 2, xcsi2rxss->pads, 0);
|         ^~~~~~~~~~~~~~~~~

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-csi2rxss.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-csi2rxss.c b/drivers/media/platform/xilinx/xilinx-csi2rxss.c
index 28724f2..33191a5 100644
--- a/drivers/media/platform/xilinx/xilinx-csi2rxss.c
+++ b/drivers/media/platform/xilinx/xilinx-csi2rxss.c
@@ -1635,7 +1635,7 @@ static int xcsi2rxss_probe(struct platform_device *pdev)
 
 	v4l2_set_subdevdata(subdev, xcsi2rxss);
 
-	ret = media_entity_init(&subdev->entity, 2, xcsi2rxss->pads, 0);
+	ret = media_entity_pads_init(&subdev->entity, 2, xcsi2rxss->pads);
 	if (ret < 0)
 		goto error;
 
-- 
1.8.5.2.233.g932f7e4

